lib/internal/test_runner/runner.js

```diff
@@ -503,6 +505,16 @@ function run(options = kEmptyObject) {
   if (only != null) {
     validateBoolean(only, 'options.only');
   }
+  if (globPatterns != null) {
+    validateArray(globPatterns, 'options.globPatterns');
```

Comment:
We should probably validate that the array is not empty. You can add the logic here. Otherwise, I will probably do it in a follow up.